packages/collector/src/announceCycle/agentready.js

```diff
@@ -142,3 +146,39 @@ function fireMonitoringEvent() {
     }
   });
 }
+
+function sendEOLEvent() {
+  agentConnection.sendEvent(
+    {
+      title: `Node.js version ${process.versions.node} reached its end of life`,
+      text:
+        'This version no longer receives updates or security fixes and might contain unfixed vulnerabilities.\n\n' +
+        'Please consider upgrading Node.js to an active version.\n\n' +
+        'For a list of active versions visit ' +
+        '[https://nodejs.org/en/about/releases/](https://nodejs.org/en/about/releases/)',
+      plugin: 'com.instana.forge.infrastructure.runtime.nodejs.NodeJsRuntimePlatform',
+      id: pidStore && typeof pidStore.getEntityId === 'function' ? pidStore.getEntityId() : undefined,
+      timestamp: Date.now(),
+      duration: ELEVEN_MINUTES,
+      severity: 5
+    },
+    err => {
+      if (err) {
+        logger.debug('Node.js version EOL', err);
+      }
+    }
+  );
+}
+
+function detectEOLNodeVersion() {
+  // This verificarion must be updated from time to time
+  // Check https://endoflife.software/programming-languages/server-side-scripting/nodejs for EOL versions
```

Comment:
The page referred to in the comment looks outdated (it still lists Node.js 10 as a current release). We should link to https://nodejs.org/en/about/releases/ instead, just as in the issue description. We could also throw in another link to https://github.com/nodejs/Release#end-of-life-releases.

An additional comment to also update the EOL check in packages/collector/test/agentCommunication_test.js when updating this one might be helpful for our future selves.

Reply:
Right, indeed this page is not reliable, and nothing is supposed to beat the official release page anyway.
I will apply the changes. Thanks!